build/android/native/jni/Android.mk

```diff
@@ -76,6 +81,8 @@ LOCAL_CFLAGS += \
 	-DUSE_FREETYPE=1                \
 	-DUSE_LEVELDB=0                 \
 	-DUSE_LUAJIT=1                  \
+	-DUSE_GETTEXT=1                 \
+	-DRUN_IN_PLACE=0                \
```

Comment:
Why is this needed?

Reply:
When testing I got a locale path set to ` ../build/locale`, I tried to remove it with this.
But after that I added the `path_locale` config in `porting_android`, so maybe this nit needed any more. Can try to remove it.

Reply:
When setting all paths to porting_android this line does nothing

Reply:
Solved.